recipes/fgap/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+echo "#!/bin/bash" >> FGAP
+ls
+echo "echo 'starting FGAP'" >> FGAP
```

Comment:
Do we really need the `echo 'starting FGAP'` here?
Couldn't we just use something like:
```bash
cat > FGAP <<EOF
#!/bin/bash
octave --path "$PREFIX/bin/" --no-gui --eval "fgap \$@"
EOF
```
?

Reply:
[Sorry that I added those comments after each other and not as part of a full review. Clicked the wrong button..]